CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- File encoding detection was inconsistent with the `sonar-scanner` engine.
```

Comment:
From a user's perspective, I don't think the problem was necessarily that the detection was inconsistent, but that it was blindly applying the encoding rather than using a heuristic. Maybe something like `Source encoding configuration was overriding the encoding detection heuristics in all cases.`?